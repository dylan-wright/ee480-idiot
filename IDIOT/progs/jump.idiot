jz $u0, $u0
sz $u0
sys
