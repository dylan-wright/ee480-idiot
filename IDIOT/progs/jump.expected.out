Text:
//generated by AIK version 20070512
@0000
8208
8201
8000
//end
Data:
//generated by AIK version 20070512
//end
